src/platform/qt/SettingsView.cpp

```diff
@@ -735,7 +735,12 @@ void SettingsView::saveSetting(const char* key, const QVariant& field) {
 	m_controller->setOption(key, field);
 	m_controller->updateOption(key);
 }
-
```

Comment:
Line break needed between functions.

Why did you add a version of this function that doesn't have a default setting? The UI is just a guideline and does not hold what the ground truth for these default should be, but with this change that's where the defaults are coming from.

Reply:
Why should the UI be just a guideline? The only example that would warrant overriding it, IMO would be fastFowardMute.

Reply:
Why would the UI not be just a guideline? It's a terrible way to view defaults since you need a visual editor or skimming through lots and lots of XML, and is several levels detached from the actual names of the settings. The actual defaults are dictated by code, and this SettingsView class doesn't even dictate what the defaults are where they're actually used: it already needs to be kept in sync. Making the UI need to be kept in sync too is just a headache and hard to debug.